compiler/luci/service/src/CircleShapeInferenceRule.cpp

```diff
@@ -1121,6 +1121,40 @@ class ShapeInferenceAlgorithm final : public luci::CircleNodeVisitor<loco::NodeS
     return loco::NodeShape{output_shape};
   }
 
+  loco::NodeShape visit(const luci::CirclePadV2 *node) final
+  {
+    const loco::DataType S32 = loco::DataType::S32;
+
+    auto input_shape = loco::shape_get(node->input()).as<loco::TensorShape>();
+    auto paddings = loco::must_cast<luci::CircleConst *>(node->paddings());
+
+    // TODO support non-const case
```

Comment:
My I think we can just return without computing anything in this case.
We can not do anything anyway.

Reply:
> without computing anything in this case.

I think this would do
```cxx
loco::NodeShape unknown;
return unknown;
```

Reply:
Or we could just return shape of the node from file
```
auto this_shape = own_shape(node);
return loco::NodeShape{this_shape};
```